src/VstsSyncMigrator.Core/Execution/MigrationContext/TestPlansAndSuitesMigrationContext.cs

```diff
@@ -1025,7 +1025,7 @@ private void InnerLog(ITestSuiteBase sourceTestSuite, string message = "", int i
                 Log.LogInformation("==      Suite Title: {SourceTestSuiteTitle}=============================".PadLeft(indent), sourceTestSuite.Title.PadRight(45));
                 Log.LogInformation("===============================================================================================".PadLeft(indent));
             }
-            Log.LogInformation("== {GetLogTags()} - suiteid[{SourceTestSuiteId}] | {message}".PadLeft(indent), sourceTestSuite.Id.ToString().PadRight(6), message);
+            Log.LogInformation($"== {GetLogTags()} - suiteid[{sourceTestSuite.Id.ToString().PadRight(6)}] | {message}".PadLeft(indent));
```

Comment:
Sry, this breaks the Serilog logging framework, it builds up a formatted log statement and injects the args into the {} based on position. But you are right there is an error here, because of not aligned with the new framework. Can you change the line to this instead and verify the output before updating the PR
`Log.LogInformation("== {Tags} - suiteid[{SourceTestSuiteId}] | {message}".PadLeft(indent), GetLogTags(), sourceTestSuite.Id.ToString().PadRight(6), message);`

Reply:
Thank you for your reply and sorry for the mistake. Personally I use NLog and so I didn't (and still don't) know how Serilog works. From what I've seen the format it uses looks a bit uncommon as it completely differs from the standard string.format and object[] args. Nevertheless, I tried your suggestion and can confirm it also fixes the exception. So I changed my PR.

Reply:
Ah the keyword is structured logging. Again what learned. Maybe that becomes helpful in my projects someday. :)